core/modules/filter/filter.module

```diff
@@ -2161,6 +2162,10 @@ function _filter_url($text, $filter) {
           if (preg_match("`<($ignore_tags)(?:\s|>)`i", $chunks[$i], $matches)) {
             $open_tag = $matches[1];
           }
+          // Check whether this tag has a class contained in $ignore_classes.
+          elseif (preg_match("`<([a-z0-9]+)\s+[\s\S]*?class=[\"'][\s\w-]*(?:$ignore_classes)[\"'\s]+(?:\s|>)`i", $chunks[$i], $matches)) {
```

Comment:
```suggestion
          elseif (preg_match("`[a-z0-9-]+\s.*class=[\"']{1}.*\b(?<!\-)$ignore_classes\b(?!\-).*[\"']{1}\s?>`i", $chunks[$i], $matches)) {
```

...see my comment in https://github.com/backdrop/backdrop-issues/issues/1926#issuecomment-653849722